Bottom: c5c0a8d77b30dce5118ede52a82222293855e48c
Top:    ada72972ae703c69e7f04146245c4890486e2a3b
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-04-25 12:13:53 -0700

Refresh of rs-iwarp

---

diff --git a/src/rsocket.c b/src/rsocket.c
index f529a3c..0d84d02 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1769,7 +1769,7 @@ static int rs_poll_cq(struct rsocket *rs)
 			if (wc.wc_flags & IBV_WC_WITH_IMM) {
 				msg = ntohl(wc.imm_data);
 			} else {
-				msg = ((uint32_t *) rs->rbuf + rs->rbuf_size)
+				msg = ((uint32_t *) (rs->rbuf + rs->rbuf_size))
 					[rs_wr_data(wc.wr_id)];
 
 			}
